GsaGH/Parameters/5_Results/PointResultGoo.cs

```diff
@@ -1,48 +1,40 @@
-﻿using Rhino.Geometry;
-using Grasshopper.Kernel;
+﻿using Grasshopper.Kernel;
 using Grasshopper.Kernel.Types;
-using System.Drawing;
-using Rhino.Display;
 using OasysUnits;
+using Rhino.Display;
+using Rhino.Geometry;
+using System.Drawing;
 
 namespace GsaGH.Parameters
 {
   public class PointResultGoo : GH_GeometricGoo<Point3d>, IGH_PreviewData
   {
-    public PointResultGoo(Point3d point, IQuantity result, Color colour, float size)
+    public PointResultGoo(Point3d point, IQuantity result, Color color, float size)
     : base(point)
     {
-      m_result = result;
-      m_size = size;
-      m_colour = colour;
+      MResult = result;
+      _mSize = size;
+      _mColor = color;
```

Comment:
```suggestion
      Result = result;
      _size = size;
      _colour = color;
```

Reply:
result and size ok, but color or colour?? XD

Reply:
review was from before our chat, agree to use _color in code